### DISCUSSION TITLE
[4b8-dev] Images in the installation of the examples of Joomla! 4

### DISCUSSION THREAD

**Discussion by Magnytu2** [joomla/joomla-cms#33189] [2. Cold Features]
Good morning all. I have been testing almost all versions of Joomla for some time now! 4. And I can never thank you enough for all your hard work. To make Joomla! 4 more modern and design how about using these images in the installation of the examples of Joomla! 4. I find the effect of his feminine faces very beautiful and above all it gives an effect of promise to Joomla !, a view of the future.

I can show you two Joomla! 4 with the two examples in line so that you can better imagine it.

![Home_J4_1](https://user-images.githubusercontent.com/26197929/115151823-ea23e600-a06e-11eb-9b17-8fd31e166143.jpg)
![Screenshot_2021-04-18 Blog](https://user-images.githubusercontent.com/26197929/115151916-4ab32300-a06f-11eb-883e-cd963a8dd9e4.png)

**Comment by richard67** [comment]
This is definitely not an issue. Moving to discussions.

**Comment by brianteeman** [comment]
What is the source and licence of those images

**Reply by Magnytu2** [reply]
These are images I found on Pixabay. https://pixabay.com/fr/users/merlinlightpainting-19833603/?tab=popular

**Reply by Magnytu2** [reply]
Pixabay License
Free for commercial use
No attribution required

**Reply by brianteeman** [reply]
Not that simple I'm afraid

**Comment by richard67** [comment]
To be honest, for me this was a useless issue and now is a useless discussion, wasting time of maintainers and contributors. One may like this kind of pictures, someone else may like that kind of pictures, but we should not try to put all these pictures who someone might like into our sample data.

For example I like cats very much, and I think there can't really be enough cats pictures in the internet, but it never came into my mind to request to put cats pictures into Joomla core of sample data.

But that's just my personal opinion, I don't speak for anybody else.

**Reply by Magnytu2** [reply]
you are right. This discussion is unnecessary in terms of Joomla code development! 4. But I would be curious to know the opinion of the communication service of Joomla. Because whether for the launch of a website or Joomla! 4. You will always be judged first by the eyes. When Joomla! 4 will be broadcast on all digital information networks, it is first and foremost the images that will speak.
And why not images of cats, that's what works best on social networks.
I'm just putting a simple idea here. I do not impose anything. You can even delete the post. I will not be offended. I only want the best for Joomla! 4. And especially that now we quickly release the next version of J4, which I hope will not be a beta.

**Reply by richard67** [reply]
The above was just my personal opinion. I'd never delete a post just because of that. Others might find it interesting.

**Reply by Magnytu2** [reply]
Especially to thank you.
![Joomla_cat](https://user-images.githubusercontent.com/26197929/115289050-75c27300-a152-11eb-96b4-f6a1ab0c8bf2.jpg)
